src/main/java/school/hei/haapi/endpoint/rest/controller/AttendanceController.java

```diff
@@ -0,0 +1,50 @@
+package school.hei.haapi.endpoint.rest.controller;
+
+
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import lombok.AllArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import school.hei.haapi.endpoint.rest.mapper.AttendanceMapper;
+import school.hei.haapi.endpoint.rest.model.AttendanceStatus;
+import school.hei.haapi.endpoint.rest.model.CreateAttendanceMovement;
+import school.hei.haapi.endpoint.rest.model.StudentAttendance;
+import school.hei.haapi.endpoint.rest.model.StudentAttendanceMovement;
+import school.hei.haapi.model.BoundedPageSize;
+import school.hei.haapi.model.PageFromOne;
+import school.hei.haapi.service.AttendanceService;
+
+@RestController
+@AllArgsConstructor
+public class AttendanceController {
+  private final AttendanceService attendanceService;
+  private final AttendanceMapper attendanceMapper;
+
+  @PostMapping("/attendance/movement")
+  public StudentAttendanceMovement createAttendanceMovement (@RequestBody CreateAttendanceMovement movement) {
+    return attendanceMapper.toRestMovement(
```

Comment:
It will look better if you go to the next line like : 
`Attendance toSave = attendanceMapper.toDomain(movement))`
`return attendanceMapper.toRestMovement(`
`attendanceService.createStudentAttendanceMovement(toSave)); `

Reply:
code refactored